apps/judicial-system/digital-mailbox-api/src/app/app.controller.ts

```diff
@@ -36,4 +44,16 @@ export class AppController {
 
     return this.appService.getCases(user.nationalId, query?.lang)
   }
+
+  @Get('case/:caseId')
+  @ApiCreatedResponse({ type: String, description: 'Get case by id' })
```

Comment:
Consider making the type here be the Case response type instead of a string so that Swagger can display a more detailed example of the response (especially if you move the digitalMailboxCase to the API and add the swagger decorators to describe the response type)